packages/ckeditor5-html-support/src/integrations/integrationutils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import type { DowncastWriter, ViewElement } from 'ckeditor5/src/engine';
+
+/**
+ * @module html-support/integrations/integrationutils
+ */
+
+/**
+ * Returns the first view element descendant matching the given view name.
+ * Includes view element itself.
+ */
```

Comment:
```suggestion
 * Includes view element itself.
 *
 * @internal
 */
```

It was private function.